css/powerup.sass

```diff
@@ -0,0 +1,22 @@
+---
+---
+main
+    background:
+        color: #222
+        image: url(/assets/img/powerup.png)
+        position: -20% 0
+        attachment: fixed
+        repeat: no-repeat
+
+main h1,
+main h5
```

Comment:
Again, most of this is identical across pages. You should split out the repetitive things into one sass file which is the same everywhere.

Reply:
(Everywhere meaning on each page that needs this code.)